packages/tooling/__tests__/lib/parameters-to-json-schema.test.js

```diff
@@ -559,11 +559,30 @@ describe('type', () => {
       ];
 
       expect(parametersToJsonSchema({ parameters })[0].schema).toStrictEqual({
-        properties: { param: { type: 'array' } },
+        properties: { param: { type: 'object' } },
```

Comment:
This test result should have always been `object`, but we were setting it to `array` due to a bug on line 242 where we'd overwrite the type we we were setting and re-using the originally supplied type (`array` in this case). 😬